mithril-aggregator/src/database/repository/signer_registration_store.rs

```diff
@@ -113,10 +105,48 @@ impl VerificationKeyStorer for SignerRegistrationStore {
 #[cfg(test)]
 mod tests {
     use crate::database::test_helper::{insert_signer_registrations, main_db_connection};
-    use crate::store::test_verification_key_storer;
+
+    use mithril_common::entities::{Epoch, PartyId, Signer, SignerWithStake};
+    use mithril_common::test_utils::fake_keys;
+    use std::collections::{BTreeMap, BTreeSet, HashMap};
+    use std::sync::Arc;
+
+    use crate::VerificationKeyStorer;
 
     use super::*;
 
+    fn build_signers(
```

Comment:
I refactor the code to simplify tests but I keep a function to create `SignerWithStake`.
The `fake_data::signers_with_stakes` is a bit long. If I use it, `signer_registration_store` tests take 0.8s instead of less than 0.2s using `fake_keys::signer_verification_key